HW6/Anastasiia-web/3.py

```diff
@@ -0,0 +1,18 @@
+# Task3. Write a function that calculates the number of characters included in a given string
+
```

Comment:
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result